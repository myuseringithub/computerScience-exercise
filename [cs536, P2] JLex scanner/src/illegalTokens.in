// illegal identifier
$name
dot.name
123
8_
1name

// illegal integer literal
99999999999999

// illegal string literal
"unterminated
2"also unterminated \"
__4"backslash followed by space: \ is not allowed"
____6"bad escaped character: \a AND not terminated
_______9"very bad string \
abc