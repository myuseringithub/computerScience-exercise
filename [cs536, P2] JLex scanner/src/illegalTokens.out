name
dot
.
name
123
8
_
1
name
2147483647
2
__4
____6
_______9
