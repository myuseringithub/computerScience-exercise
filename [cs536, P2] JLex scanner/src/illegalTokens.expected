  2: 2	name
  3: 1	dot
  3: 4	.
  3: 5	name
  4: 1	123
  5: 1	8
  5: 2	_
  6: 1	1
  6: 2	name
  9: 1	2147483647
  9:16	324234
  9:22	.
  9:23	234234234
 13: 1	2
 14: 1	__4
 15: 1	____6
 16: 1	_______9
